b2/console_tool.py

```diff
@@ -344,6 +344,25 @@ def run(self, args):
         return 0
 
 
+class GetFileInfoAndDeleteFileVersion(Command):
```

Comment:
Good point with the completion. Since we have `download_file_by_id`, so we could have `delete_by_id` / `delete_file_by_id` / `delete_file_id` too.